cpp/util/etcd.cc

```diff
@@ -669,18 +592,21 @@ void EtcdClient::WatchState::RequestDone(GenericResponse* gen_resp,
 
   VLOG(2) << "etcd_index: " << gen_resp->etcd_index;
 
-  // TODO(alcutter): doing this here works around some etcd 401 errors, but in
-  // the case of sustained high qps we could miss updates entirely (e.g. if
-  // this new index is already past the 1000 entry horizon by the time we make
-  // the new watch request.)  One way to address this might be to have the
-  // watcher re-do an "initial" get on the target, and, in the case of
-  // directory watches, maintain a set of known keys so that it can synthesise
+  // TODO(alcutter): doing this here works around some etcd 401
```

Comment:
Did clang-format get all aggressive on wrapping this comment? looks like there's loads of space left on these lines !